spec/request/messages_request_spec.rb

```diff
@@ -170,22 +163,21 @@
     end
   end
 
-  context 'with a user that has not accepted T&C' do
-    let(:mhv_account) do
-      double('mhv_account',
-             accessible?: false,
-             eligible?: true,
-             needs_terms_acceptance?: true,
-             upgraded?: false)
+  context 'with a user that is not a va patient', :skip_mvi do
+    let(:mhv_account_type) { 'Premium' }
+    let(:current_user) { build(:user, :mhv, mhv_account_type: mhv_account_type) }
```

Comment:
Doesn't setting `:mhv` already set the `mhv_account_type` to `'Premium'`, or you're just being explicit here?